packages/material-ui/src/Chip/Chip.js

```diff
@@ -38,7 +38,7 @@ export const styles = theme => {
       verticalAlign: 'middle',
       boxSizing: 'border-box',
       '&$disabled': {
-        opacity: 0.5,
+        opacity: theme.palette.action.disabledOpacity,
```

Comment:
Don't handwave this change. We need this in a separate pull request. Potentially during v5. The  impact is hard to anticipate in a lab release.

Reply:
OK, I will stick to the usage in the Pagination.

Reply:
This is also definitely a feature we should highlight. The theme should be your first place to start customizing (following the systems design approach).